app/controllers/application_controller.rb

```diff
@@ -14,6 +14,10 @@ def set_slimmer_headers
     response.headers[Slimmer::Headers::TEMPLATE_HEADER] = "header_footer_only"
   end
 
+  def finder_base_path
+    "/#{finder_slug}"
```

Comment:
Is it a bit weird that we take the slug from the `params` and then add a slash back in front of it for the `base_path`?